server/lib/retry_on_conflict.coffee

```diff
@@ -0,0 +1,25 @@
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+promises_ = __.require 'lib', 'promises'
+error_ = __.require 'lib', 'error/error'
+
+module.exports = (updateFn, maxAttempts = 10)-> (args...)->
```

Comment:
this lib is aspiring to a greater purpose than its current entity based implementation (as it has been wisely put in the general lib folder), making it the most readable possible would be a nice, like expliciting args name in an object 🤓

Reply:
that's the beauty of it: as it's generalized, we don't know what the arguments will be :D thus this name of `args` to pass it to the function on line 13 without knowing anything about it

Reply:
oops i was talking of `updateFn, maxAttempts` to turn it into an object, exactly the beauty your talking about for the `args` object

Reply:
addressed by 80bec96